packages/rocketchat-livechat/.app/client/views/message.js

```diff
@@ -40,7 +41,7 @@ Template.message.helpers({
 			case 'wm':
 				return t('Welcome', { user: this.u.username });
 			case 'livechat-close':
-				return t('Conversation_finished');
+				return (!_.isEmpty(Livechat.conversationFinishedMessage)) ? Livechat.conversationFinishedMessage : t('Conversation_finished');
```

Comment:
if possible I would try to not use underscore =)

Reply:
Fixed.